src/Altinn.AccessManagement/Controllers/RightsController.cs

```diff
@@ -227,5 +228,45 @@ public async Task<ActionResult<RightsDelegationResponseExternal>> Delegation([Fr
                 return new ObjectResult(ProblemDetailsFactory.CreateProblemDetails(HttpContext, detail: "Internal Server Error"));
             }
         }
+
+        /// <summary>
+        /// Gets a list of all recipients having received right delegations from the reportee party including the resource/app/service info, but not specific rights
+        /// </summary>
+        /// <param name="input">Used to specify the reportee party the authenticated user is acting on behalf of. Can either be the PartyId, or the placeholder values: 'person' or 'organization' in combination with providing the social security number or the organization number using the header values.</param>
+        /// <param name="cancellationToken">Cancellation token used for cancelling the inbound HTTP</param>
+        /// <returns>A <see cref="Task{TResult}"/> representing the result of the asynchronous operation.</returns>
+        [Authorize(Policy = AuthzConstants.POLICY_ACCESS_MANAGEMENT_READ)]
+        [ActionName("ListOfferedDelegations")]
+        [HttpGet("{party}/rights/delegation/offered")]
+        [Produces(MediaTypeNames.Application.Json, Type = typeof(IEnumerable<RightDelegationExternal>))]
+        [ProducesResponseType(typeof(ProblemDetails), StatusCodes.Status400BadRequest)]
+        [ProducesResponseType(typeof(void), StatusCodes.Status401Unauthorized)]
+        [ProducesResponseType(typeof(void), StatusCodes.Status403Forbidden)]
+        [ProducesResponseType(typeof(ProblemDetails), StatusCodes.Status500InternalServerError)]
+        public async Task<IActionResult> GetDelegationOfferedList([FromRoute, FromHeader] RightDelegationOfferedInput input, CancellationToken cancellationToken)
```

Comment:
Endpoint should be behind the same feature toggle as delegation and delegationcheck until go-live:
[FeatureGate(FeatureFlags.RightsDelegationApi)]